pepper-apis/src/main/java/org/broadinstitute/ddp/route/ListStudyLanguagesRoute.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.ddp.route;
+
+import org.apache.http.HttpStatus;
+import org.broadinstitute.ddp.constants.ErrorCodes;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dao.StudyLanguageDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.json.errors.ApiError;
+import org.broadinstitute.ddp.util.ResponseUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class ListStudyLanguagesRoute implements Route {
+    private static final Logger LOG = LoggerFactory.getLogger(ListStudyLanguagesRoute.class);
+
+    @Override public Object handle(Request request, Response response) throws Exception {
+        String studyGuid = request.params(RouteConstants.PathParam.STUDY_GUID);
+        LOG.debug("Received request for supported languages for study {}", studyGuid);
```

Comment:
Nice! Rule-of-thumb is for every route to start with some logging for the request. Can you turn this into `LOG.info()`?